pkg/hintrunner/zero/zerohint_ec.go

```diff
@@ -11,11 +11,83 @@ import (
 	secp_utils "github.com/NethermindEth/cairo-vm-go/pkg/hintrunner/utils"
 	"github.com/NethermindEth/cairo-vm-go/pkg/utils"
 	VM "github.com/NethermindEth/cairo-vm-go/pkg/vm"
+	"github.com/NethermindEth/cairo-vm-go/pkg/vm/memory"
 	mem "github.com/NethermindEth/cairo-vm-go/pkg/vm/memory"
 	"github.com/consensys/gnark-crypto/ecc/stark-curve/fp"
 	"github.com/holiman/uint256"
 )
 
+// BigIntToUint256 hint guesses the low part of the result uint256 variable
+//
+// `newBigIntToUint256Hint` takes 2 operanders as arguments
+//   - `low` is the variable that will store the low part of the uint256 result
+//   - `x` is the BigInt variable to convert to uint256
+func newBigIntToUint256Hint(low, x hinter.ResOperander) hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "BigIntToUint256",
+		Op: func(vm *VM.VirtualMachine, ctx *hinter.HintRunnerContext) error {
+			//> ids.low = (ids.x.d0 + ids.x.d1 * ids.BASE) & ((1 << 128) - 1)
+
+			lowAddr, err := low.GetAddress(vm)
+			if err != nil {
+				return err
+			}
+
+			xAddr, err := x.GetAddress(vm)
+			if err != nil {
+				return err
+			}
+
+			xBigInt, err := vm.Memory.ResolveAsBigInt3(xAddr)
+			if err != nil {
+				return err
+			}
+
+			var xD0Big big.Int
+			(*xBigInt[0]).BigInt(&xD0Big)
+
+			var xD1Big big.Int
+			(*xBigInt[1]).BigInt(&xD0Big)
```

Comment:
Did you mean `(*xBigInt[1]).BigInt(&xD1Big)` here

Reply:
oh yeah... how is it possible it works like this? aha

Reply:
ok i see , `BigIntToUint256` was imported in the cairo file but not used ><